index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <section class="topbar">
+    <div class="logo">Abdul Nganje</div>
+    <div class="menu-icon"><img src="images/Union.svg" alt=""></div>
+    </section>
+    <section class="headline">
+    <h1 class="title">Hey There.<br>I’m Nganje</h1>
+    <h2 class="subtitle">I’am a Software Developer</h2>
+    <p class="description">I can help you build a product , feature or website Look 
+    through some of my work and experience! If you like what you see and have a 
+    project you need coded, don’t hestiate to contact me.</p>
+    <div class="social-media">
+    <div class="github"><img src="images/GitHub.svg" alt=""></div>
+    <div class="linkedin"><img src="images/Vector.svg" alt=""></div>
+    <div class="chill"><img src="images/Vector1.svg" alt=""></div>
+    <div class="twitter"><img src="images/Vector2.svg" alt=""></div>
+    <div class="medium"><img src="images/Vector3.svg" alt=""></div>
+    </div>
```

Comment:
- Your list of icons should use a ul tag to group them, and li tag per icon. Each icon should have a link (a tag)